dist/tools/testrunner/testrunner.py

```diff
@@ -26,9 +26,7 @@ def find_exc_origin(exc_info):
     pos = list_until(extract_tb(exc_info),
                      lambda frame: frame[0].startswith(PEXPECT_PATH)
                     )[-1]
-    return pos[3], \
-           os.path.relpath(os.path.abspath(pos[0]), RIOTBASE), \
-           pos[1]
+    return (pos[3], os.path.relpath(os.path.abspath(pos[0]), RIOTBASE), pos[1])
```

Comment:
If you put this in one line, the parenthesis are not needed

Reply:
>If you put this in one line, the parenthesis are not needed

the parens aren't needed even for multiple lines. See current state (:

Reply:
The current state works because of `\`

Reply:
You can get rid of them on multiple lines if you put parenthesis (that would be my preferred solution) but since we may allow 120 characters line length (see #8141), it's fine to have it in one line

Reply:
I know they are not needed, but from my quick google survey it seemed to me like they are good practice and don't hurt, isn't that the case?!

Reply:
For me it's ok, I won't block the PR just for that ;)